python/ray/resource_spec.py

```diff
@@ -213,15 +214,25 @@ def resolve(self, is_head, node_ip_address=None):
             memory = (avail_memory - object_store_memory - (redis_max_memory
                                                             if is_head else 0))
             if memory < 100e6 and memory < 0.05 * system_memory:
-                raise ValueError(
-                    "After taking into account object store and redis memory "
-                    "usage, the amount of memory on this node available for "
-                    "tasks and actors ({} GB) is less than {}% of total. "
-                    "You can adjust these settings with "
-                    "ray.init(memory=<bytes>, "
-                    "object_store_memory=<bytes>).".format(
-                        round(memory / 1e9, 2),
-                        int(100 * (memory / system_memory))))
+                # Linux, BSD has cached memory, which should
+                # also be considered as unused memory
+                memory += cached_memory
```

Comment:
According to https://gitlab.com/procps-ng/procps/-/blob/master/proc/sysinfo.c#L805  , how add cached_memory/2 here?